Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -274,10 +291,17 @@ def network_create_engine_wrapper(self, pstate, initialdlstatus=None):
 
                 atp["ti"] = torrentinfo
                 has_resume_data = resume_data and isinstance(resume_data, dict)
+
+                if has_resume_data is not None:
```

Comment:
`has_resume_data` is of type bool, you should not check whether this variable is None or not.